platforms/js/build_js.py

```diff
@@ -269,6 +279,11 @@ def build_doc(self):
         if check_file(opencvjs_test_path):
             log.info("OpenCV.js tests location: %s", opencvjs_test_path)
 
+    if args.build_perf:
+        opencvjs_perf_path = os.path.join(builder.build_dir, "bin", "perf", "base.js")
+        if check_file(opencvjs_perf_path):
+            log.info("OpenCV.js tests location: %s", opencvjs_test_path)
```

Comment:
It's not really important, but it should be `opencvjs_perf_path`

Reply:
Thanks for feedback! Track it in #276

Reply:
Thanks @terfendail for review. I clicked the merge button too quickly. In next two weeks, I will be on vacation. @Wenzhao-Xiang , please work closely with Vitaly on the PR review. Thanks all!

Reply:
Get it! Have a nice time for your trip!

Reply:
Have a good vacation!